.github/workflows/contracts-admin.yml

```
@@ -0,0 +1,54 @@
+name: Build & Push Contracts
+
+on:
+  push:
+    tags: ["contracts-admin-v*.*.*"]
+
+jobs:
+  release:
+    name: Build & Push Contracts
+    runs-on: ubuntu-latest
+    env:
+      DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
+      DOTNET_CLI_TELEMETRY_OPTOUT: 1
+    steps:
+      - name: Version
+        id: version
+        run: |
+          VERSION=${GITHUB_REF#refs/tags/contracts-admin-v}
+          if ! [[ ${VERSION} =~ ^[0-9]+\.[0-9]+\.[0-9]+(-[a-zA-Z0-9\.]+)?$ ]]
+          then
+            echo 1>&2 "Invalid version: ${VERSION}"
+            exit 1
+          fi
+          echo "version=${VERSION}" >> $GITHUB_OUTPUT
+      - name: Checkout
+        uses: actions/checkout@v2
+        with:
+          lfs: true
+      - name: Setup .NET SDK
+        uses: actions/setup-dotnet@v3
+        with:
+          dotnet-version: 7.0.x
+      - name: Restore
+        working-directory: src/LeanCode.Contracts.Admin
+        run: dotnet restore
+      - name: Generate Protobuf
+        working-directory: src/LeanCode.Contracts.Admin
+        run: SERVER_VERSION=2.0.0-alpha.4 ../../tools/generate.sh project --project=LeanCode.Contracts.Admin.csproj -o LeanCode.Contracts.Admin.pb
```

I think it should be
```suggestion
        run: SERVER_VERSION=2.0.0-alpha.4 ../../tools/generate.sh project --project=LeanCode.Contracts.Admin.csproj -o LeanCode.Contracts.pb
```